src/backgrounds/_base.scss

```diff
@@ -0,0 +1,113 @@
+/**
+ * @file Backgrounds
+ * @module Backgrounds
+ * @overview Background and matching text styles for page modules
+ * @todo: remove _backdrops.scss since this replaces it?
+ * @todo: rename TrialWarranty to be generic, update to use Starry Background
```

Comment:
I do think this kinda removes the need for TrialWarranty being its own thing, since it's pretty much a callout, but if it's easier for people to just drop it in with the right styling tweaks, maybe it can live